filc/cmds.go

```diff
@@ -361,6 +361,7 @@ var retrieveFileCmd = &cli.Command{
 
 			selspec, err := textselector.SelectorSpecFromPath(
 				dmSelText,
+				true,
```

Comment:
@ribasushi is this right?

Reply:
95% it is right, will open a new PR if this turns out to be a problem ( need to actually test this live to confirm )